backend/src/test/java/zipgo/review/application/ReviewServiceTest.java

```diff
@@ -124,4 +134,38 @@ class ReviewServiceTest extends ServiceTest {
                 .isInstanceOf(MemberException.NotFound.class);
     }
 
+    @Test
+    void 리뷰를_수정할_수_있다() {
+        //given
+        PetFood 식품 = 키워드_없이_식품_초기화(브랜드_조회하기());
+        Member 멤버 = memberRepository.save(무민());
+        petFoodRepository.save(식품);
+        Review 리뷰 = reviewRepository.save(혹평_리뷰_생성(멤버, 식품,
+                List.of(눈물_이상반응().getName(), 먹고_토_이상반응().getName())));
+
+        //when
+        reviewService.updateReview(멤버.getId(), 리뷰.getId(), 리뷰_수정_요청());
+
+        //then
+        Review 저장된_리뷰 = reviewRepository.getById(리뷰.getId());
+        assertAll(
+                () -> assertThat(저장된_리뷰.getRating()).isEqualTo(4),
+                () -> assertThat(저장된_리뷰.getComment()).isEqualTo("change comment"),
+                () -> assertThat(저장된_리뷰.getTastePreference()).isEqualTo(EATS_MODERATELY),
+                () -> assertThat(저장된_리뷰.getStoolCondition()).isEqualTo(UNCERTAIN),
+                () -> assertThat(저장된_리뷰.getAdverseReactions()).hasSize(1)
+        );
+    }
+
+    @Test
+    void 잘못된_리뷰_id로_리뷰를_수정하면_예외_처리() {
+        //given
+        키워드_없이_식품_초기화(브랜드_조회하기());
+        Member 멤버 = memberRepository.save(무민());
+
+        //when, then
+        Assertions.assertThatThrownBy(() -> reviewService.updateReview(멤버.getId(), 123456789L, 리뷰_수정_요청()))
```

Comment:
이건 그냥 제 팁인데요! 저는 검증부 가독성을 높이기 위해 이렇게 변수를 선언해주는 편입니다~
```suggestion
        var 잘못된_리뷰_id = 123456789L;
        Assertions.assertThatThrownBy(() -> reviewService.updateReview(멤버.getId(), 잘못된_리뷰_id, 리뷰_수정_요청()))
```